### ISSUE TITLE
Jellyfin server error

### ISSUE THREAD

**Issue by BlayZin** [liaralabs/swizzin#374]
# Please note that issues that do not follow the format to some degree can be automatically closed.

## Bug description
When I want to reach the Jellyfin website from the dashboard I get an Error message

## Reproduction steps
1. Install package Jellyfin
2. Go to Panel and click Jellyfin
3. Get "Error processing request." on blank page

## Installed applications through swizzin
nginx, panel, rutorrent, rtorrent, medusa, radarr, plex, tautulli, jackett

## Expected behavior
It should open the Jellyfin dashboard.

## Server Info
**(please complete the following information):**
 - OS: Ubuntu 18.04
 - Image Source: installscript from Hetzner

## Is Swizzin up to date?
- Please run `box update`, and report whether that was succesful. Please try to reproduce your issue again.

- Please mention the commit hash you are seeing
  - [e.g. `HEAD is now at `**`d4d151b`** `...`]

It is on ab973ee

## Additional context
I first installed Jellyfin on 89615e9 and it worked and now with latest commits I upgraded with "box upgrade jellyfin" and then it wasn't reachable anymore.
I looked in the default config from nginx but the `server_name` is the right one but it is not possible to reach it with the domain.
The server IP on the other hand works.
Other Services like Medusa are still reachable.

Good sources for log output could be the following: (only if they have relevant information)
- `cat /root/logs/install.log`
- `cat /root/logs/swizzin.log`
- `cat /var/logs/nginx/error.log`
- `systemctl status <app>@<(user)>`
- `journalctl -xe`

**Please paste your code into code blocks using the codeblock format like so**

Systemctl Log
```
jellyfin.service - Jellyfin
   Loaded: loaded (/etc/systemd/system/jellyfin.service; enabled; vendor preset: enabled)
   Active: active (running) since Tue 2020-07-21 16:01:10 CEST; 1min 30s ago
 Main PID: 19338 (jellyfin)
    Tasks: 15 (limit: 4915)
   CGroup: /system.slice/jellyfin.service
           └─19338 /opt/jellyfin/jellyfin -d /home/blayzin/.config/Jellyfin

Jul 21 16:01:28 blayz.in jellyfin[19338]: [16:01:28] [ERR] [4] Emby.Server.Implementations.HttpServer.HttpListenerHost: Error processing request. URL: https://d.blayz.in/jellyfin
Jul 21 16:01:28 blayz.in jellyfin[19338]: System.FormatException: An invalid IP address was specified.
Jul 21 16:01:28 blayz.in jellyfin[19338]:  ---> System.Net.Sockets.SocketException (22): Invalid argument
Jul 21 16:01:28 blayz.in jellyfin[19338]:    --- End of inner exception stack trace ---
Jul 21 16:01:28 blayz.in jellyfin[19338]:    at System.Net.IPAddressParser.Parse(ReadOnlySpan`1 ipSpan, Boolean tryParse)
Jul 21 16:01:28 blayz.in jellyfin[19338]:    at System.Net.IPAddress.Parse(String ipString)
Jul 21 16:01:28 blayz.in jellyfin[19338]:    at Emby.Server.Implementations.Networking.NetworkManager.IsInPrivateAddressSpace(String endpoint, Boolean checkSubnets)
Jul 21 16:01:28 blayz.in jellyfin[19338]:    at Emby.Server.Implementations.Networking.NetworkManager.IsInPrivateAddressSpace(String endpoint)
Jul 21 16:01:28 blayz.in jellyfin[19338]:    at Emby.Server.Implementations.HttpServer.HttpListenerHost.ValidateHost(String host)
Jul 21 16:01:28 blayz.in jellyfin[19338]:    at Emby.Server.Implementations.HttpServer.HttpListenerHost.RequestHandler(IHttpRequest httpReq, String urlString, String host, String localPath, CancellationToken cancellationToken)
```

**Comment by liaralabs** [comment]
This is a regression with how Local Addresses are handled in 10.6. Related issue: https://github.com/jellyfin/jellyfin/issues/3607

Workaround for now -- in /home/username/.config/Jellyfin/config/system.xml:

Replace:
```
  <LocalNetworkAddresses>
    <string>0.0.0.0</string>
  </LocalNetworkAddresses>
```
with
```
  <LocalNetworkAddresses />
```
And restart jellyfin with `systemctl restart jellyfin`

**Comment by liaralabs** [comment]
The PR in question has been merged to mainline and is available in 10.6.1.
